lib/backend/services/user_service.dart

```diff
@@ -518,4 +518,19 @@ class UserServiceCentralized implements UserService {
     final UserData _userData = UserData.fromJson(_responseMap);
     return _userData;
   }
+
+  Future<void> deleteUser(String userAddress, String password) async {
+    final Map<String, dynamic> _body = <String, String>{
+      'password': password,
+    };
```

Comment:
password is not required (cc: @jdeepee was supposed to fix this in docs)

Reply:
Cool, will remove